custom_components/adjustable_bed/beds/okin_cb24.py

```
@@ -0,0 +1,395 @@
+
+def build_cb24_command(command_value: int, bed_selection: int = 0x00) -> bytes:
+    """Build a 7-byte CB24 protocol command.
+
+    Args:
+        command_value: 32-bit integer representing the command
+        bed_selection: Bed selection byte (0x00=default, 0xAA=bed A, 0xBB=bed B)
+
+    Returns:
+        7-byte command: [0x05, 0x02, cmd3, cmd2, cmd1, cmd0, bed_selection]
+    """
+    return bytes([0x05, 0x02] + int_to_bytes(command_value) + [bed_selection])
```

Ruff (0.14.14)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)